src/app/views/provisioning/order.controller.coffee

```diff
@@ -72,6 +71,13 @@ angular.module 'mnoEnterpriseAngular'
         )
         .finally(() -> vm.isLoading = false)
     else
+      currenciesArray = []
+      _.forEach(vm.subscription.product.pricing_plans,
+        (pp) -> _.forEach(pp.prices, (p) -> currenciesArray.push(p.currency)))
+
+      vm.currencies = _.uniq(currenciesArray)
+      vm.filteredPricingPlans = if vm.subscription?.product?.pricing_plans then vm.filterCurrencies() else []
```

Comment:
we don't need the conditional anymore because we've already checked if we have a cached subscription.